platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -1023,8 +1023,9 @@ class ProjectSchemaAccessorsIntegrationTest : AbstractKotlinIntegrationTest() {
         )
 
         executer.expectDocumentedDeprecationWarning("The Project.getConvention() method has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#deprecated_access_to_conventions")
-        executer.expectDocumentedDeprecationWarning("The org.gradle.api.plugins.Convention type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#deprecated_access_to_conventions")
-
+        (1..2).forEach {
```

Comment:
The additional deprecation comes from:

https://github.com/gradle/gradle/blob/783e4aa305d675d10e2f5f56f2c5794d15356689/platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/accessors/runtime/Runtime.kt#L45

Should we change that access so it does not trigger deprecations at runtime?

Reply:
Done